Marlin/src/lcd/extui/lib/mks_ui/draw_cloud_bind.h

```diff
@@ -0,0 +1,38 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2020 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+
+#ifdef __cplusplus
+extern "C" { /* C-declarations for C++ */
+#endif
```

Comment:
Is there a reason for `extern "C"`?

It seems like all the functions are declared and defined in these files, so I expect C++ linkage should be fine.

Reply:
This applies to other headers as well.

Reply:
Some of the functions declared here are used inside the MKS lvl lib (dependency of this code). And the LVGL is pure C code. That's the reason for the `extern "C"`. But we didn't verify what part is really used and part part not, we just let it.